kik_unofficial/datatypes/xmpp/chatting.py

```diff
@@ -12,8 +12,15 @@
 from lxml.etree import Element
 
 from kik_unofficial.datatypes.peers import Group
-from kik_unofficial.datatypes.xmpp.base_elements import XMPPElement, XMPPResponse, XMPPContentResponse, \
-    XMPPReceiptResponse, XMPPOutgoingContentMessageElement, XMPPOutgoingMessageElement, XMPPOutgoingIsTypingMessageElement
+from kik_unofficial.datatypes.xmpp.base_elements import (
```

Comment:
Same here

Reply:
The drawback of using formatters. I can force it to not format it, but that will add 
`# fmt: off` and `# fmt: on` around it